hw/ip/otbn/dv/otbnsim/sim/state.py

```diff
@@ -58,25 +58,35 @@ def __init__(self) -> None:
         self._new_rnd_data = None  # type: Optional[int]
         self._urnd_reseed_complete = False
 
+    @property
+    def pc_next(self) -> int:
+        if self._pc_next_override:
+            return self._pc_next_override
+        return self.pc + 4
```

Comment:
As in previous commits, please can we just make this an explicit function (`get_next_pc` maybe?)